app/views/katello/layouts/katello.html.erb

```diff
@@ -4,11 +4,13 @@
 
 <% content_for(:stylesheets) do %>
   <%= stylesheet "katello/katello" %>
+  <%= stylesheet_link_tag *webpack_asset_paths('katello', :extension => 'css'), "data-turbolinks-track" => true %>
 <% end %>
 
 <% content_for(:javascripts) do %>
   <%= javascript_include_tag "katello/common/vendor" %>
   <%= javascript_include_tag "katello/common" %>
+  <%= javascript_include_tag *webpack_asset_paths('katello', :extension => 'js'), "data-turbolinks-track" => true, 'defer' => 'defer' %>
```

Comment:
imho this should not be deferred, just the part that does the mounting. i assume it should speed up loading.

Reply:
@ohadlevy I'm not sure where else it would go? When I remove the `defer` from here then the js tries to execute before the DOM node is available.

Reply:
well, your code should not be doing any mounting... if you remove the reactDom mounting part, then you can in a script tag do something like 
```js
<script defer="defer">
  $(tfm.reactMounter.mount('SomeReactElement', '#element', {some: 'data'}));
</script>
```

Reply:
These changes are no longer required with the new approach in https://github.com/Katello/katello/pull/7032

Reply:
@waldenraines it seems like we'll still need to defer since the React code is executed before the `div#content` element is rendered, right?